src/scanners/javascript.js

```diff
@@ -90,31 +84,26 @@ export default class JavaScriptScanner {
       // the scan. See: https://github.com/mozilla/addons-linter/issues/1288
       ignore: false,
       patterns: ['!bower_components/*', '!node_modules/*'],
-      // Also, don't ignore dotfiles in scans.
-      dotfiles: true,
```

Comment:
@willdurand did you checked if we do have coverage to be sure that dotfiles are not ignored in the scans?

Reply:
Good call, let me double check.